platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -477,9 +478,9 @@ private void wire() {
         // to move the logic of SavedStateController::markSavedState into the signedStateFileManager. There is no reason
```

Comment:
This big `FUTURE_WORK` comment is no longer relevant

Reply:
This comment is actually wrong I think. The reason this logic is not in the `signedStateFileManager` is because state saving can take minutes. And in that time, the queue can build up with states that do not need to be saved to disk. The comment should reflect that so someone else doesn't get confused in the future.

Reply:
Removed the comment here. Added a comment to `SavedStateController`. I think this sort of thing is more specific to the component and doesn't really belong where we wire things together.

```
/**
 * Controls which signed states should be written to disk based on input from other components.
 * <p>
 * This is intentionally split out of the component that actually saves the state. Saving the state can take a long
 * time, and we don't want the queue of states waiting to be handled by that component to grow large while we are
 * waiting to write a state. It is much better simply not to put states into that component's queue if we don't want to
 * write them to disk.
 */
public interface SavedStateController {
```